src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+public class Intake extends SubsystemBase {
+    private static Intake INSTANCE;
+    private final WPI_TalonSRX motor = new WPI_TalonSRX(Ports.Intake.MOTOR);
+    private final Solenoid solenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Intake.SOLENOID);

```

Comment:
1) Usually when we have pneumatics, instead of calling a solenoid "solenoid", we call them by their function. In this case, "retractor" will fit nicely.
2) Isn't that a double solenoid?

Reply:
It isnt a double solenoid, name will be changed